zubhub_frontend/zubhub/src/views/login/Login.jsx

```diff
@@ -1,273 +1,148 @@
-import React from 'react';
-import clsx from 'clsx';
-import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
-
 import { connect } from 'react-redux';
-
-import { makeStyles } from '@mui/styles';
-import Visibility from '@mui/icons-material/Visibility';
-import VisibilityOff from '@mui/icons-material/VisibilityOff';
+import * as AuthActions from '../../store/actions/authActions';
 import {
-  Grid,
   Box,
-  Divider,
-  Container,
-  Card,
-  CardActionArea,
-  CardContent,
-  Typography,
+  Grid,
+  FormControl,
+  FormLabel,
+  OutlinedInput,
   InputAdornment,
+  Typography,
   IconButton,
-  OutlinedInput,
-  InputLabel,
-  FormHelperText,
-  FormControl,
+  FormControlLabel,
+  Checkbox
 } from '@mui/material';
+import { CustomButton, CustomErrorMessage } from '../../components';
+import { TfiLock, TfiArrowLeft } from 'react-icons/tfi';
+import { SlEye, SlUser } from 'react-icons/sl';
+import { BsEyeSlash } from 'react-icons/bs';
+import { makeStyles } from '@mui/styles';
+import { useState } from 'react';
+import { mainStyles, step2Styles } from '../../assets/js/styles/views/login/loginStyles';
+import { handleLogin, validationSchema } from './loginScripts';
+import { useFormik } from 'formik';
 
-import {
-  validationSchema,
-  handleClickShowPassword,
-  handleMouseDownPassword,
-  login,
-} from './loginScripts';
-
-import { withFormik } from 'formik';
-
-import CustomButton from '../../components/button/Button';
-import * as AuthActions from '../../store/actions/authActions';
-import styles from '../../assets/js/styles/views/login/loginStyles';
-const useStyles = makeStyles(styles);
+const useMainStyles = makeStyles(mainStyles);
+const useStyles = makeStyles(step2Styles);
 
-/**
- * @function Login View
- * @author Raymond Ndibe <ndiberaymond1@gmail.com>
- *
- * @todo - describe function's signature
- */
-function Login(props) {
+const Login = props => {
+  const [{ password }, setShowPassword] = useState({ password: false });
   const classes = useStyles();
-
-  const [state, setState] = React.useState({
-    show_password: false,
-  });
-
-  const handleSetState = obj => {
-    if (obj) {
-      Promise.resolve(obj).then(obj => {
-        setState(state => ({ ...state, ...obj }));
-      });
-    }
-  };
-
-  const { show_password } = state;
-  const { t } = props;
+  const mainClasses = useMainStyles();
+  const {
+    errors,
+    handleChange,
+    handleBlur,
+    handleSubmit,
+    touched,
+  } = useFormik({
```

Comment:
unless the way we were using formik before is no longer working, please use it the same way